<launch>
    <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" >
        <param name="marker_size"           type="double" value="7.0" />
        <param name="max_new_marker_error"  type="double" value="0.08" />
        <param name="max_track_error"       type="double" value="0.2" />
        <param name="output_frame_from_msg" type="bool" value="false" />
        <param name="output_frame" type="string" value="/reference/head_camera" />

        <param name="marker_frame_prefix"   type="string" value="color_tracker_" />
        <remap from="camera_image"  to="/io/internal_camera/head_camera/image_raw" />
        <remap from="camera_info"   to="/io/internal_camera/head_camera/camera_info" />  
    </node>


</launch>
